docs/guides/rate-limits.mdx

```diff
@@ -162,58 +162,89 @@ Endpoint based rate limits are controls applied to individual API endpoints with
+### Purpose of endpoint-based rate limits
 
-These limits act as a first line of defense for your project endpoints. They analyze incoming request patterns and consider
-factors such as:
-
-- Source IP Address: Identifies and potentially blocks requests from suspicious sources or those exhibiting behavior indicative of
-  malicious activity.
-- Request Frequency: Monitors how often requests are made to a specific endpoint to detect and thwart attempts to overwhelm the
-  system or exploit vulnerabilities.
-- User Authentication: (If applicable) Considers whether requests are authenticated and may apply different limits for
-  authenticated vs. unauthenticated requests.
-- Request Method: May apply different limits based on the HTTP method used (GET, POST, etc.).
-- IP Whitelist Status: Applies higher limits to whitelisted IPs for Enterprise and Growth customers.
-
-### Purpose endpoint based rate limits
-
-Ory implements endpoint based rate limits to proactively secure individual endpoints and protect against common attack vectors
+Ory implements endpoint-based rate limits to proactively secure individual endpoints and protect against common attack vectors
 like brute-force and credential stuffing, while allowing for higher volumes of legitimate traffic from trusted sources. These
 attacks typically involve numerous attempts to guess credentials or exploit vulnerabilities, often from a limited set of IP
 addresses.
 
-1. Enhanced security:
-
+1. **Enhanced security:**
    - Restricts the number of requests from specific sources within a given timeframe.
    - Makes it significantly harder for attackers to succeed with brute-force or credential stuffing attacks.
    - Strengthens the security of your Ory projects and protects sensitive user data.
 
-2. Protection against malicious bots:
-
+2. **Protection against malicious bots:**
    - Differentiates between genuine user traffic and potentially harmful bot activity.
    - Analyzes request patterns to identify and block automated malicious activities.
 
-3. Safeguarding specific endpoints:
-
+3. **Safeguarding specific endpoints:**
    - Offers granular control over how each endpoint handles traffic and responds to potential threats.
    - Allows fine-tuning of security measures for individual endpoints.
    - Optimizes protection without compromising the user experience.
 
-4. Fair usage:
-
+4. **Fair usage:**
    - Complements project rate limits in ensuring fair resource allocation.
    - Contributes to a fairer and more stable platform for all users by mitigating abusive traffic.
 
-5. Flexibility for high volume legitimate traffic:
+5. **Flexibility for high-volume legitimate traffic:**
    - Provides options for Enterprise and Growth customers to whitelist internal IPs for higher rate limits.
    - Balances security needs with the requirements of high-volume legitimate traffic.
 
-### Notes on rate limit rules
+### Types of endpoint-based protection
+
```

Comment:
I think here we should keep the distinction of layers:
1. Global CDN layer with Cloudflare
2. Local Business logic layer with Envoy
The current proposal flattens that which i don't think is a good idea.

Reply:
I intentionally chose not to disclose these "implementation" details, as they shouldn't affect the customer, and it also allows us the flexibility to modify the implementation without needing to update this documentation or notify the customers.